src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useReducer, useEffect } from 'react';
+import {
+  Stepper, FullscreenModal, Container, Button,
+} from '@edx/paragon';
+import PropTypes from 'prop-types';
+import {
+  setHighlightStepperModal,
+} from '../data/actions';
+import { initialStepperModalState, stepperModalReducer } from '../data/reducer';
+import HighlightStepperTitle from './HighlightStepperTitle';
+import HighlightStepperSelectCourses from './HighlightStepperSelectCourses';
+import HighlightStepperConfirmCourses from './HighlightStepperConfirmCourses';
+import HighlightStepperConfirmHighlight from './HighlightStepperConfirmHighlight';
+import HighlightStepperFooterHelpLink from './HighlightStepperFooterHelpLink';
+
+const ContentHighlightStepper = ({ openModal }) => {
+  /* Stepper Modal currently accessible from:
+  - ContentHighlightSetCard.jsx
+  - CurrentContentHighlightHeader.jsx
+  - ZeroStateHighlights.jsx
+  */
+  /* eslint-disable no-unused-vars */
+  const [stepperModalState, dispatch] = useReducer(stepperModalReducer, initialStepperModalState);
+  const steps = ['Title', 'Select courses', 'Confirm and Publish', 'All Set'];
+  const [currentStep, setCurrentStep] = useState(steps[0]);
+  const [modalState, setModalState] = useState(openModal);
+  useEffect(() => {
+    setModalState(openModal);
+  }, [openModal]);
+  const submitAndReset = () => {
+    if (steps.indexOf(currentStep) === steps.length - 1) {
+      /* submit data to api if confirmed */
```

Comment:
nice note for our future selves :) nit: add in a `TODO` so its easy to search for as well, e.g. `TODO: submit data to api if confirmed`